ckan/model/package.py

```diff
@@ -527,17 +527,40 @@ def activity_stream_item(self, activity_type, revision, user_id):
                 activity_type = 'deleted'
 
         try:
-            d = {'package': dictization.table_dictize(self,
-                context={'model': ckan.model})}
-            return activity.Activity(user_id, self.id, revision.id,
-                    "%s package" % activity_type, d)
+            # We save the entire rendered package dict so we can support
+            # viewing the past packages from the activity feed.
+            dictized_package = ckan.logic.get_action('package_show')({
+                'model': ckan.model,
+                'session': ckan.model.Session,
+                'for_view': True,
```

Comment:
for_view is a thing that makes me sad. IIRC it will do things like translate strings into the current user's language which doesn't seem appropriate for storing as a permanent record in the activity. Is there a reason it's required here?